plugins/package-managers/gleam/src/main/kotlin/GleamManifest.kt

```diff
@@ -29,7 +29,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 internal data class GleamManifest(
     /** List of all resolved packages including transitive dependencies. */
-    val packages: List<Package>
+    val packages: List<Package> = emptyList()
```

Comment:
It seems we're not calling the constructor at all. 

In my view, the choice for / against using a default value would be based on whether the property is mandatory or optional in upstream data model. Omitting the default value would make the deserialization fail in case the property is not present.

What do you think?

Reply:
I see. I stated it wrong, I meant more like, it does not simplify any call (at least according to this diff).